server/lib/csp.js

```diff
@@ -25,17 +25,22 @@ function addCdnRuleIfRequired(target) {
   return target;
 }
 
-function isCspRequired(req) {
+function isCspRequired(path) {
   // is the user running tests? No CSP.
-  return req.path !== '/tests/index.html';
+  return path !== '/tests/index.html' &&
+         ! /\.(js|json|css|woff|ttf|eot|svg|mustache)$/.test(path);
```

Comment:
The simple answer is only HTML pages should have CSP headers. There is probably a better test, but I wasn't sure what it was.